service/src/java/org/apache/hive/service/cli/thrift/ThriftHttpServlet.java

```diff
@@ -289,7 +287,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
       } else {
         try {
           LOG.error("Login attempt is failed for user : " +
-              getUsername(request) + ". Error Messsage :" + e.getMessage());
+              httpAuthService.getUsername(request) + ". Error Messsage :" + e.getMessage());
         } catch (Exception ex) {
           // Ignore Exception
```

Comment:
Are we aware of any exceptions happening here? if the goal is to keep doPost method signature free from exceptions, we should throw a:
```
new RuntimeException(ex)
```
ignore it is always dangerous

Reply:
This was existing code, but I did the changes that you suggested. 
I don't know what was the goal of it because the method `throws Exception`.

Reply:
With this change that I did of throwing an exception in this place a few unit tests started to fail.
In this place, the exception used to be ignored in case its logging had failed.
In any case, the response status is set properly to reflect an error
`response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);`
I reverted throwing of the exception and instead improved the comment with explanation of why it's ignored.
Do you find it's ok?

Reply:
ack